### PR TITLE
Added telemetry properties for MORE SUBSYSTEMS!!!!!

### PR DESCRIPTION
**Pull request by imathno**
Fixed indentation and conventions in AgitatorManagerSubsystem

### CODE REVIEW

File: Competition2017/src/competition/subsystems/agitator/AgitatorSubsystem.java [hunk 1]
```diff
@@ -5,20 +5,21 @@
...
-public class AgitatorSubsystem extends BaseSubsystem {
+public class AgitatorSubsystem extends BaseSubsystem implements PeriodicDataSource {
 
     protected final XCANTalon agitatorMotor;
     private final RobotSide side;
     protected final DoubleProperty intakePowerProperty;
     protected final DoubleProperty ejectPowerProperty;    
 
-    public AgitatorSubsystem(
+    public AgitatorSubsystem(
```
Comment (WasabiFan) [inline Competition2017/src/competition/subsystems/agitator/AgitatorSubsystem.java]: Remove trailing whitespace

**Review by WasabiFan — APPROVED** [review]

**Comment by WasabiFan** [comment]
It'd be good to make a quick pass over the other subsystems and make sure that this is everywhere.

**Comment by aschokking** [comment]
Good point, in particular it seems like we'd like to see the Climber motor stats in the short-run

**Review by aschokking — APPROVED** [review]

**Comment by WasabiFan** [comment]
Rename the PR now that you expanded the changes

**Comment by WasabiFan** [comment]
When you squash+merge this, let's _not_ include the emoji :wink:

**Pull request merged**